sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -231,6 +231,7 @@ class SparkSession private(
   /**
    * A collection of methods for registering user-defined data sources.
    */
+  @Unstable
```

Comment:
We should either remove `@Unstable` or `private[sql]` if we want this as an API.

Reply:
I'll make it `private[sql]` for now before we decide to expose this as an API.